### PR TITLE
Enrich partial documentation

### PR DESCRIPTION
**Pull request by benzen**
Hope this could help this project to gain more visibility.

Don't hesite to propose correction.

### CODE REVIEW

File: features.md [hunk 1]
```diff
@@ -184,11 +184,127 @@ public class App {
...
+                .__() //div
+            .__() //form```
+)
+```
+
+This way of invoking partial is particularly useful when you need to use a smaller part (component) gathered together to produce a bigger template.
+This is the most common usage of partials.
+
+There is another way of using partials, it's to construct a layout. The layout is a normal template, but with a hole to be filed with partials.
+Like we saw earlier, a partial has nothing special by himself. What is interesting is the layout, consider the following template.
```
Comment (fmcarvalho) [inline features.md:248]: `by itself` and not `himself` (I think so, but not sure)

File: features.md [hunk 2]
```diff
@@ -184,11 +184,127 @@ public class App {
...
+So we have defined our layout, let's use it to create a template with much less clutter.
+
+```
+public class APage {
+
+    public static DynamicHtml<Object> view = (DynamicHtml<Object>) DynamicHtml.view(Layout::template, Object model, new DynamicHtml<Object>[]{Menu::template, APage::template]).threadSafe();
+
+    private static <T> void template(DynamicHtml<T> view, T model, HtmlView[] partials) {
+        view
+            .div().text("Type safty feel so cozy !!")
```
Comment (fmcarvalho) [inline features.md:302]: Typo in "safty"

**Comment by benzen** [comment]
Both of them are corrected

**Comment by fmcarvalho** [comment]
WoW it's great !!!! 

In truth sometimes it is much harder to me to be in the other side of the library and look to it as a client and not only the author. 
Your descriptions are very meaningful.

I have already left 2 comments.

And maybe this section could be renamed to "Partials and Layouts" or something else, because you have given very useful insights about the use of layouts too.

Also, if you wish, after these small fixes and I merge your PR you could make a small PR to replace the section "_Partial Views_" from the master README.md with an highlighted link to your section in documentation.

Thank you

**Comment by benzen** [comment]
Done

**Review by fmcarvalho — APPROVED** [review]

**Comment by benzen** [comment]
It's pretty normal to feel that way, I assume you've worked quite a long time on this.
It's far more easier for me to look for what I want to achieve, and just report about it in the documentation :)

**Pull request merged**